tests/features/custom_page/custom_page.search_field.feature

```diff
@@ -0,0 +1,38 @@
+@api
+Feature: "Custom page" search.
+  In order to use the search
+  As a user of the website
+  I need to be able to view search results in the "Custom page" canonical page.
+
+  Background:
+    Given the following collection:
+      | title  | Code camp  |
```

Comment:
Don't reuse existing strings for new tests. "Code camp" belongs to the Add Custom Page feature.